keps/sig-node/1769-memory-manager/README.md

```diff
@@ -666,20 +628,13 @@ details). For now, we leave it here.
 _This section must be completed when targeting beta graduation to a release._
 
 * **How does this feature react if the API server and/or etcd is unavailable?**
+  No impact.
 
 * **What are other known failure modes?**
-  For each of them, fill in the following information by copying the below template:
-  - [Failure mode brief description]
-    - Detection: How can it be detected via metrics? Stated another way:
-      how can an operator troubleshoot without logging into a master or worker node?
-    - Mitigations: What can be done to stop the bleeding, especially for already
-      running user workloads?
-    - Diagnostics: What are the useful log messages and their required logging
-      levels that could help debug the issue?
-      Not required until feature graduated to beta.
-    - Testing: Are there any tests for failure mode? If not, describe why.
+  Not applicable.
```

Comment:
the enable, disable, enable flow appears to have some gotchas.  How does an operator identify that it is  happening?

Reply:
the kubelet restart will fail with the appropriate error message under the error under the kubelet log

Reply:
> the kubelet restart will fail with the appropriate error message under the kubelet log

so this will first show up as a node readiness failure and then they'll be looking at kubelet logs?  Please add the detail to the PRR answers.